lib/python/treadmill_aws/cli/admin/aws/instance.py

```diff
@@ -143,7 +148,11 @@ def create(
         if not key:
             key = metadata.instance_keys()[0]
 
-        hostnames = hostmanager.create_host(
+        if hostname and (count > len(hostname)):
```

Comment:
should it be rather mutually exclusive? either count, or hostnames? probably ok for now, but i think this interface is becoming bit confusing.